### PR TITLE
Add LeitorDeManga

### LINKED ISSUE #2251: Leitor de Mangá - Source Request
### Source name

Leitor de Mangá

### Source link

https://leitordemanga.com

### Source language

portuguese

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by choppeh**
Closes #2251

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/pt/leitordemanga/build.gradle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ext {
+    extName = 'Leitor de mangá'
```
Comment (vetleledaal) [inline src/pt/leitordemanga/build.gradle]: Why not `Leitor de Mangá`? The site uses that over the non-capitalized mangá.
If you change it, you should also change it in the `.kt` file.

File: src/pt/leitordemanga/res/mipmap-xxxhdpi/ic_launcher.png
Comment (vetleledaal) [inline src/pt/leitordemanga/res/mipmap-xxxhdpi/ic_launcher.png:1]: Looks very much like the JP flag, maybe use a different color than white? The original logo has a transparent background.

File: src/pt/leitordemanga/build.gradle (hunk 1 above)
Comment (choppeh) [reply src/pt/leitordemanga/build.gradle]: Resoved: https://github.com/keiyoushi/extensions-source/pull/2280/commits/d103e792cb35d0a7bc93c96cd83860cabf62c098

File: src/pt/leitordemanga/res/mipmap-xxxhdpi/ic_launcher.png
Comment (choppeh) [reply src/pt/leitordemanga/res/mipmap-xxxhdpi/ic_launcher.png:1]: I made a custom icon to follow the webpage theme

**Review by vetleledaal — DISMISSED** [review]

File: src/pt/leitordemanga/src/eu/kanade/tachiyomi/extension/pt/leitordemanga/LeitorDeManga.kt
Comment (AwkwardPeak7) [inline src/pt/leitordemanga/src/eu/kanade/tachiyomi/extension/pt/leitordemanga/LeitorDeManga.kt:1]: the `mangaSubString` should be set to `ler-manga`; 

`https://leitordemanga.com/ler-manga/`

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]

File: src/pt/leitordemanga/src/eu/kanade/tachiyomi/extension/pt/leitordemanga/LeitorDeManga.kt
Comment (choppeh) [reply src/pt/leitordemanga/src/eu/kanade/tachiyomi/extension/pt/leitordemanga/LeitorDeManga.kt:1]: I only saw the suffix "manga" at the end. I need glasses :smile: 

And it was working with "manga" as a substring.

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**